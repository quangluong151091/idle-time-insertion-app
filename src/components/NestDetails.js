import React from 'react';
import Stickers from '../components/Stickers';
import NestStatus from '../components/NestStatus';

class NestDetails extends React.Component {
    handleClick = () => {
        this.props.history.push('/')
    }
    
    render() {
        return (
            <div className="content">
                <button className="back-button" onClick={this.handleClick}><i className="fas fa-home"/></button>
                <table className="users">
                    <tbody>
                        <tr>
                        <Stickers />
                        <NestStatus name={this.props.location.pathname.substr(1)} />
                        </tr>
                    </tbody>
                </table>
            </div>
        );
    }
}

export default NestDetails;