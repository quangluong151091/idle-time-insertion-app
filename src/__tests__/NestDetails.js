import React from 'react';
import { shallow } from 'enzyme';

import NestDetails from '../components/NestDetails';

describe('NestDetails', () => {
  it('should render correctly', () => {

    const component = shallow(<NestDetails location={{'pathname': "/29484001"}}/>);
  
    expect(component).toMatchSnapshot();
  });
});