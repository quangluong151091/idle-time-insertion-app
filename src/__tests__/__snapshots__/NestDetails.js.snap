// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NestDetails should render correctly 1`] = `
<div
  className="content"
>
  <button
    className="back-button"
    onClick={[Function]}
  >
    <i
      className="fas fa-home"
    />
  </button>
  <table
    className="users"
  >
    <tbody>
      <tr>
        <Stickers />
        <NestStatus
          name="29484001"
        />
      </tr>
    </tbody>
  </table>
</div>
`;
